nameko/cli/run.py

```diff
@@ -168,7 +169,14 @@ def main(args):
             import_service(path)
         )
 
-    config = {AMQP_URI_CONFIG_KEY: args.broker}
+    config = {}
+
+    if args.config:
+        with open(args.config) as fle:
+            config = yaml.load(fle)
+    else:
+        config[AMQP_URI_CONFIG_KEY] = args.broker
```

Comment:
Let's keep ``--broker`` for backwards compatibility. @davidszotten already raised the notion of registering config keys in https://github.com/onefinestay/nameko/issues/223. We can track these ideas there.